### PR TITLE
front: stdcm: add available simulation div

### LINKED ISSUE #8056: front: stdcm: add available simulation message
The message should appear when the simulation has finished loading, and it disappears when the user interacts with one of the elements of the proposed result, or when he modifies the form inputs.

Image

### PR DESCRIPTION
**Pull request by Akctarus**
closes #8056

### CODE REVIEW

File: front/src/applications/stdcmV2/views/StdcmViewV2.tsx [hunk 1]
```diff
@@ -20,26 +22,42 @@ const StdcmViewV2 = () => {
...
-      {stdcmV2Results?.stdcmResponse && !isPending && (
-        <StdcmResults
-          // TODO: Next step : use currentSimulationInputs instead of stdcmSimulationResults to handle multiples simulations results
-          stdcmData={stdcmV2Results?.stdcmResponse}
-          pathProperties={pathProperties}
-        />
-      )}
+      <div
+        className={`stdcm-container ${!isPending && !currentSimulationInputs && !interactedResultsElements ? 'simulation-visible' : ''}`}
+      >
```
Comment (Uriel-Sautron) [inline front/src/applications/stdcmV2/views/StdcmViewV2.tsx]:
```suggestion
     <div
        className={cx('stdcm-container', {
          'simulation-visible':
            !isPending && !currentSimulationInputs && !interactedResultsElements,
        })}
      >
```

**Review by Uriel-Sautron — APPROVED** [review]
LGTM !! Tested just a comment

File: front/src/applications/stdcmV2/views/StdcmViewV2.tsx (hunk 1 above)
Comment (Akctarus) [reply front/src/applications/stdcmV2/views/StdcmViewV2.tsx]: Nice catch ! done

**Review by kmer2016 — APPROVED** [review]
Well done ! Tested

**Pull request merged**